tests/foreman/performance/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Test utilities for writing foreman/performance tests
+All test cases for performance tests are defined in this module
```

Comment:
The first portion of a docstring should be a summary that is no more than one line long. It looks like this:

    """My summary line.

    My details. Details can span
    many lines.

    They can be as verbose as I want. They're totally different
    than the summary line.

    """